narwhals/_compliant/typing.py

```diff
@@ -154,3 +155,16 @@ class ScalarKwargs(TypedDict, total=False):
     "Callable[[CompliantFrameT, WindowInputs[NativeExprT]], Sequence[NativeExprT]]"
 )
 """A function evaluated with `over(partition_by=..., order_by=...)`."""
+
+NarwhalsAggregation: TypeAlias = Literal[
+    "sum", "mean", "median", "max", "min", "std", "var", "len", "n_unique", "count"
+]
+"""`Expr` methods we aim to support in `DepthTrackingGroupBy`.
+
+Be sure to update me if you're working on one of these:
```

Comment:
😂 I am sure the type checker would complain anyway but good to mention issues as well